sklearn/model_selection/tests/test_split.py

```diff
@@ -1345,9 +1282,9 @@ def test_cv_iterable_wrapper():
                             list(kf_randomized_iter_wrapped.split(X, y)))
 
     try:
+        splits_are_equal = True
         np.testing.assert_equal(list(kf_iter_wrapped.split(X, y)),
                                 list(kf_randomized_iter_wrapped.split(X, y)))
-        splits_are_equal = True
```

Comment:
Line change doesn't seem to be necessary?

Reply:
This way both lines are covered, but the test remains the same.